### PR TITLE
[Term Entry] PyTorch Tensors .rand()

### LINKED ISSUE #4938: [Term Entry] PyTorch Tensors .rand()
### Reason/inspiration (optional)

We would like a new entry on the `.rand()` term under PyTorch `Tensors` concept. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/rand/rand.md`.

The entry should include:

- An introduction to the concept
- A ‘Syntax’ section that provides the syntax for the concept (if applicable)
- An ‘Example’ section that provides an example demonstrating the concept in use
- A ‘Codebyte Example’ section that provides a codebyte example demonstrating the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by deenovita**
Fixes #4938

### Description

- Added rand term.
- Close issue #4938  

### Type of Change

- Adding a new term entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/rand/rand.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Meanwhile, `torch.rand()` have other seven optional arguments that can be used to customize the tensor if needed.
+
+- `generator` is used for generating number for sampling.
+- `out`, to store the output into existing tensor (if there is any).
+- `dtype` is used to customize the data type of the output.
+- `layout`, to specify how Tensor is stored in memory.
+- `device`, to specify where the tensor will be created.
+- `requires_grad`, to record the operation of the tensor.
+- `pin_memory` is used for allocating the tensor in the pinned memory.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/rand/rand.md:41]: Modifications:

- Let's change the sentences a bit
- Let's convert commas after the parameters into colons
- Let's change the parameter descriptions a bit

```suggestion
The `.rand()` function takes one required parameter `size`, which represents the shape of the tensor that is to be created. It can create a tensor with one, two, or more dimensions.

Meanwhile, there are seven other optional parameters that can be used with it to customize the tensor if needed:

- `generator`: Used for generating a number for sampling.
- `out`: Stores the output into existing tensor (if there is any).
- `dtype`: Used to customize the data type of the output.
- `layout`: Specifies how the tensor is stored in the memory.
- `device`: Specifies where the tensor will be created.
- `requires_grad`: Records the operation of the tensor.
- `pin_memory`: Used for allocating the tensor in the pinned memory.
```

File: content/pytorch/concepts/tensors/terms/rand/rand.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Example
+
+The example below demonstrates how the `size` parameter can be applied to create three different types of tensors. `tensor1D` is a one-dimensional tensor, or vector, with a size of 3; `tensor2D` is a two-dimensional tensor, typically called a matrix; and `tensor3D` is a three-dimensional tensor.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/rand/rand.md:45]: Let's change the introduction for the example a bit:

```suggestion
The example below demonstrates how the `size` parameter can be applied to the `.rand()` function to create three different types of tensors. Among these tensors, `tensor1D` is a one-dimensional tensor or vector with a size of 3; `tensor2D` is a two-dimensional tensor, typically called a matrix; and `tensor3D` is a three-dimensional tensor:
```

File: content/pytorch/concepts/tensors/terms/rand/rand.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+print(tensor1D)
+
+# Create a two-dimensional tensor of size five by five.
+tensor2D = torch.rand(5, 5)
+print(tensor2D)
+
+# Create a three-dimensional tensor of three by five by five tensor
+tensor3D = torch.rand(3, 5, 5)
+print(tensor3D)
+
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/rand/rand.md:61]: Let's change the code block a bit:

```suggestion
import torch

# Create a one-dimensional tensor of size 3
tensor1D = torch.rand(3)
print(tensor1D)

# Create a two-dimensional tensor of size 5 by 5
tensor2D = torch.rand(5, 5)
print(tensor2D)

# Create a three-dimensional tensor of size 3 by 5 by 5
tensor3D = torch.rand(3, 5, 5)
print(tensor3D)
```

File: content/pytorch/concepts/tensors/terms/rand/rand.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+         [0.3292, 0.3411, 0.0896, 0.2175, 0.4013]],
+
+        [[0.2818, 0.3043, 0.1920, 0.2259, 0.5281],
+         [0.2549, 0.5776, 0.6986, 0.3627, 0.7794],
+         [0.7877, 0.7343, 0.9829, 0.8692, 0.5039],
+         [0.7503, 0.5880, 0.6443, 0.8743, 0.0187],
+         [0.9316, 0.5449, 0.4706, 0.1604, 0.7445]]])
+```
+
+Though one- and two-dimensional tensors are well-known, there are instances in machine learning where a three-dimensional tensor is essential.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/rand/rand.md:106]: Modifications:

- Let's merge the outputs into a single output
- Let's change the introduction for the output a bit
- Let's move the note from before the output to after the output
- Let's start the note with `> Note:` as notes should be started this way
- Let's change the note text and the last sentence a bit

````suggestion
Here is the output for the above example:

```shell
tensor([0.4581, 0.8923, 0.9110])

tensor([[0.9166, 0.4592, 0.1805, 0.5239, 0.4623],
        [0.8937, 0.9080, 0.9411, 0.5834, 0.8225],
        [0.4368, 0.9289, 0.7732, 0.7876, 0.5329],
        [0.2059, 0.1736, 0.4619, 0.4613, 0.2558],
        [0.7025, 0.7887, 0.0750, 0.7822, 0.7054]])

tensor([[[0.7478, 0.4375, 0.9769, 0.3897, 0.9407],
         [0.9649, 0.6459, 0.7927, 0.1902, 0.9625],
         [0.1942, 0.4772, 0.5795, 0.5631, 0.1460],
         [0.9176, 0.5342, 0.1067, 0.1989, 0.9478],
         [0.2999, 0.2278, 0.8297, 0.4244, 0.2445]],

        [[0.0626, 0.1151, 0.2487, 0.0667, 0.9877],
         [0.2304, 0.8230, 0.7293, 0.6631, 0.9981],
         [0.3059, 0.0967, 0.4872, 0.8695, 0.2324],
         [0.6734, 0.6645, 0.8859, 0.0637, 0.7380],
         [0.3292, 0.3411, 0.0896, 0.2175, 0.4013]],

        [[0.2818, 0.3043, 0.1920, 0.2259, 0.5281],
         [0.2549, 0.5776, 0.6986, 0.3627, 0.7794],
         [0.7877, 0.7343, 0.9829, 0.8692, 0.5039],
         [0.7503, 0.5880, 0.6443, 0.8743, 0.0187],
         [0.9316, 0.5449, 0.4706, 0.1604, 0.7445]]])
```

> Note: The result may not be exactly the same as the one displayed above, due to random number generation.

Though one and two-dimensional tensors are well-known, there are instances in machine learning where a three-dimensional tensor is essential.
````

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @deenovita, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Good job, @deenovita!

This entry looks good for a second round of review. 👍
